### ISSUE TITLE
ARIA & Screen Reader Accessibility

### ISSUE THREAD

**Issue by chenejac** [vivo-project/VIVO#4099]
**Is your feature request related to a problem? Please describe.**
VIVO should be aligned with AA level of WCAG regarding *AIRA and Screen reader accessibility**.
